docs/implementation-guide.md

```diff
@@ -82,48 +99,48 @@ gh version 2.14.7 (2022-08-25)
 https://github.com/cli/cli/releases/tag/v2.14.7
 ```
 
-### Step 3: Fork the phdi-google-cloud Repository
-Fork the phdi-google-cloud repository into your organization's, or your personal, GitHub account.
-1. Navigate to [https://github.com/CDCgov/phdi-google-cloud](https://github.com/CDCgov/phdi-google-cloud).
+### Step 3: Fork the phdi-azure Repository
+Fork the phdi-azure repository into your organization's, or your personal, GitHub account.
+1. Navigate to [https://github.com/CDCgov/phdi-azure](https://github.com/CDCgov/phdi-azure).
 2. Click on the `Fork` button in the top right.
 ![fork-repo-1](./images/fork-repo-1.png)
 3. Select the owner of the forked repository. We recommend that this is an organization and not an individual.
 4. Click `Create fork` at the bottom of the page.
 ![fork-repo-2](./images/fork-repo-2.png)
 
 ### Step 4: Clone the Forked Repository
-Clone the forked version of the phdi-google-cloud repository by running `git clone https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-google-cloud.git`. If you do not have `git` installed please follow [this guide](https://github.com/git-guides/install-git) to install it.
+Clone the forked version of the phdi-azure repository by running `git clone https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-azure.git`. If you do not have `git` installed please follow [this guide](https://github.com/git-guides/install-git) to install it.
 
 ### Step 5: Run the Quick Start Script
-In this step we will work through GCP's [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) to grant your phdi-google-cloud repo access to deploy the pipelines to your organization's GCP environment. We have provided a script to automate most of this process that we recommend you use. However, if you prefer to work through it manually you may follow [this guide](https://github.com/google-github-actions/auth#setup).
+In this step we will work through Azure's [Workload Identity Federation](https://learn.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation) to grant your phdi-azure repo access to deploy the pipelines to your organization's Azure environment. We have provided a script to automate most of this process that we recommend you use. However, if you prefer to work through it manually you may follow [this guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azcli).
 
 From your machine's command line:
 1. Navigate to the root directory of the repository you cloned in step 4.
 2. Run the quick start script and follow the prompts.
     - [quick-start.sh](../quick-start.sh) for Mac and Linux
     - [quick-start.ps1](../quick-start.ps1) for Windows
```

Comment:
Have we tested the power shell version yet? If not maybe this is something @BradySkylight could tackle next week.